native/src/modules/endpoint/reducers/pushPoi.js

```diff
@@ -14,7 +14,7 @@ const pushPoi = (state: CityContentStateType, action: PushPoiActionType): CityCo
   const getPoiRoute = (): PoiRouteStateType => {
     const allAvailableLanguages = new Map(cityLanguages.map(lng => [lng.code, null]))
     if (!path) {
-      return { status: 'ready', path: null, models: pois, allAvailableLanguages, language, city }
+      return { routeType: 'poi', status: 'ready', path: null, models: pois, allAvailableLanguages, language, city }
```

Comment:
I am kind of confused that the logic for news and pois is that much simpler then for events and categories :thinking: